hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/impl/TestContainerDataYaml.java

```diff
@@ -188,22 +189,19 @@ public void testCreateContainerFileWithoutReplicaIndex(
     cleanup();
   }
 
-
   @ContainerLayoutTestInfo.ContainerTest
-  public void testIncorrectContainerFile(ContainerLayoutVersion layout)
-      throws IOException {
+  public void testIncorrectContainerFile(ContainerLayoutVersion layout) {
     setLayoutVersion(layout);
-    try {
-      String containerFile = "incorrect.container";
-      //Get file from resources folder
+    String containerFile = "incorrect.container";
+
+    IllegalArgumentException exception = assertThrows(IllegalArgumentException.class, () -> {
+      // Get file from resource folder
       ClassLoader classLoader = getClass().getClassLoader();
       File file = new File(classLoader.getResource(containerFile).getFile());
```

Comment:
nit: these lines can be moved outside of `assertThrows`, and then it can use a single statement instead of a block.

Reply:
Done.